components/common/posts/PostsFeed.tsx

```diff
@@ -465,23 +472,46 @@ function PostControls({
   item: AppBskyFeedDefs.FeedViewPost
   onReport?: (uri: string) => void
 }) {
+  const { data: workspaceList } = useWorkspaceList()
+  const { mutate: addToWorkspace } = useWorkspaceAddItemsMutation()
+  const { mutate: removeFromWorkspace } = useWorkspaceRemoveItemsMutation()
+  const isInWorkspace = workspaceList?.find((uri) => uri === item.post.uri)
```

Comment:
Just for convenience!
```suggestion
  const isInWorkspace = workspaceList?.includes(item.post.uri)
```